docs/manage/configuration.md

```diff
@@ -170,6 +170,13 @@ Components of the same model are supported using the same low-level code.
 - `attributes`: A struct to define things like how the component is wired to the robot, its dimensions, and other specifications; attributes vary widely between models.
   See the [component documentation](/components/) for a given component type and model for more details.
 
+{{% alert title="Tip" color="tip" %}}
+
+Some optional attributes have default values.
+If you omit these attributes from your config, or if you include them but leave their values empty, RDK will apply the default values even though the default values are not shown in the config.

```

Comment:
```suggestion
If you omit these attributes from your config, or if you include them but leave their values empty, the RDK will apply the default values even though the default values are not shown in the config.
```

Also not sure whether it's are not shown or are not in the config

Reply:
I've always thought it was "RDK" without "the." This isn't part of our style guide--maybe we should nail it down one way or the other?
I don't understand the question; I guess they're not in the config but they sort of are in a way behind the scenes maybe? But the point is that the user won't see them configured, which is what I meant to convey here.

Reply:
(Have heard other people say "RDK" without an article as well, but maybe that's just a Bend office thing??)

Reply:
If we want RDK to evoke SDK, we should consider matching its usage: `the SDK`, `our SDK`, `the Python SDK`. Omitting the article is perhaps more appropriate for named products, like `ROS` or `RoboDevoKito`.

I don't mind either personally, as long as we are consistent throughout. Sounds like a check in with Product? Sounds like you were thinking the same thing!

`are not shown` -- in this case, I often see the phrase `are not reflected` to indicate that the user won't see the values where they otherwise might expect them. So maybe: `the RDK will apply these default values, even though they are not reflected in the configuration shown under the **Config** tab.` or something similar.

Reply:
Asking!

I like that suggestion, or maybe "...will apply these default values at runtime, even though they are not reflected in the configuration file"

Reply:
```suggestion
If you omit these attributes from your config, or if you include them but leave their values empty, `viam-server` will apply these default values at runtime, even though they are not reflected in the configuration file.
```
@mcvella Since you're part of that Slack thread bringing you in. Does it make more sense to say `viam-server` or the RDK in here?

Reply:
Why not just say:

If you omit these attributes from your config, or if you include them but leave their values empty, default values will be applied at runtime, even though they are not reflected in the configuration file.

Reply:
Jessamy looking for `RDK` vs `the RDK` guidance!

Reply:
Well, I didn't say it that way because that would be passive voice, which we try to avoid to reduce vagueness.

Reply:
I can just apply
```suggestion
If you omit these attributes from your config, or if you include them but leave their values empty, default values will be applied at runtime, even though they are not reflected in the configuration file.
```
but similar cases appear throughout the docs so I do think this conversation is broadly applicable and therefore worthwhile!
If it is the case that how default values are applied is actually super complicated/technical and can't be accurately described by saying that `viam-server` does it, then perhaps passive voice really is the best option here, which would be good to know for this and other similar cases.

Reply:
Ooh per Slack conversation, will use `viam-server`